### PR TITLE
Modify fancy arrow parameter for link mixin

### PR DESCRIPTION
**Pull request by edwardsfriedman**
For “back” arrow instead of “forward”

### CODE REVIEW

**Comment by josephco** [comment]
@edwardsfriedman these changes look good to me. Maybe @misscs will want to verify before merging into core.

**Comment by edwardsfriedman** [comment]
@josephco thanks! @misscs whaddaya think?

**Comment by ckeisato** [comment]
@edwardsfriedman Just checking but you did test this with the links that currently use the `fancy` attribute right?

File: base/typography/_links_config.scss [hunk 1]
```diff
@@ -35,5 +36,13 @@ $active: '&:active';
...
       margin-left: 2px;
     }
+  } @else if $fancy-before {
+    &:before {
+      content: "‹";
+      color: inherit;
+      font-style: normal;
+      line-height: 1;
+      margin-right: 2px;
+    }
```
Comment (misscs) [inline base/typography/_links_config.scss]: There's a lot of duplication here. What if we combine the shared properties into one ruleset?

**Comment by misscs** [comment]
Once this is merged in, you'll need to follow [these steps for tagging](https://github.com/CasperSleep/nightshade-core/blob/master/contributing.md#tagging-your-release). Youre release would be a minor update. 

One casper end, you will need to update the version in shrinkwrap. Running `npm install caspersleep/nightshade-core -S` should take care of updating everything. Holler if you have any issues.

File: base/typography/_links_config.scss (hunk 1 above)
Comment (misscs) [inline base/typography/_links_config.scss]: Another option could be to have one param `$fancy` and have it accept either `before` or `after`. And then run the checks on whether before or after specified. You could interpolate the value for the psedo-selector

``` sass
   &:#{$fancy} {

     @if $fancy = before {
     content: "‹";
      margin-right: 2px;
    } @else { } 
      color: inherit;
      font-style: normal;
      line-height: 1;
    }


```

**Comment by edwardsfriedman** [comment]
@ckeisato yep!

**Comment by edwardsfriedman** [comment]
@misscs Made the changes! so it only accepts three params and expects "before" or "after" as the third. If the parameter is null or a string other than "before" or "after", no arrow will appear.

**Comment by misscs** [comment]
Noice! This iteration is ++

**Pull request merged**